pombola/feedback/views.py

```diff
@@ -27,6 +31,10 @@ def add(request):
             # if there is any content in the honeypot field then label this comment as spammy
             if form.cleaned_data['website']:
                 feedback.status = 'spammy'
+
+            # if the recaptcha fails label the comment as spammy
+            if not recaptcha_client.verify(recaptcha_response):
+                feedback.status = 'spammy'
```

Comment:
I think instead of telling the user their feedback was successfully saved, we ought to tell them it wasn't saved. Otherwise there might be legitimate submissions rejected for some reason while we're telling them it's received, making it less likely that people will continue participating.

It could even be something like "Sorry, something went wrong. Please try again or email us at 
contact@pa.org.za"

![Screenshot_2020-09-17_17-22-34](https://user-images.githubusercontent.com/235801/93492432-17f35180-f90b-11ea-8dc7-b42a18ab97ae.png)

Reply:
could you add a screenshot to the PR description of this update working?